setup.cfg

```diff
@@ -23,6 +23,7 @@ install_requires =
     pydantic ~= 1.7
     pynwb
     PyYAML
+    numpy
```

Comment:
note: even though pynwb is likely to pull in numpy, since imported explicitly in our code indeed better to have it